smarts/core/waypoints.py

```diff
@@ -19,27 +19,31 @@
 # THE SOFTWARE.
 import queue
 import random
-import math
-from typing import Sequence
+import warnings
+from collections import defaultdict, namedtuple
 from dataclasses import dataclass
-from collections import namedtuple, defaultdict
-from scipy.interpolate import interp1d
+from typing import Sequence
+
 import numpy as np
+from scipy.interpolate import interp1d
 
-import warnings
+from smarts.core.utils.file import supress_pkg_resources
 
 with warnings.catch_warnings():
     warnings.filterwarnings("ignore", "numpy.ufunc size changed")
-    from sklearn.neighbors import KDTree
+    with supress_pkg_resources():
+        # We force sklearn to use a different importer since sklearn's use of pkg_resources is
+        # aggressive
+        from sklearn.neighbors import KDTree
 
 from .coordinates import Heading, Pose
 from .utils.math import (
```

Comment:
Perhaps use full path instead of relative path import?
```suggestion
from smarts.core.coordinates import Heading, Pose
from smarts.core.utils.math import (
```